keps/sig-storage/20190130-csi-raw-block-support.md

```diff
@@ -162,3 +163,5 @@ and we can test with them
 * Separated NodeStage / NodePublish calls.
 * Fixed volume reconstruction after kubelet restart.
 * Stress tests as noted above.
+
+### K8s 1.18: GA
```

Comment:
Add a note that 1.18 is the target? Same below

Reply:
Hey @msau42 @jsafrane can you link to which KEP issue#s this relates to so we can better track? 

Thanks!

Reply:
There is https://github.com/orgs/kubernetes-csi/projects/4, not sure it should be part of the KEP though.

Reply:
@msau42, not sure I understood, changed to `K8s 1.18: Targeting GA`

Reply:
I guess there's a title for this section, but no content under it. Is there anything we should be adding to the section? More test cases/corner cases to fix/test?

Reply:
Added this:
> * Disruptive testing with block devices in /dev reordered after reboot.